README.md

```diff
@@ -14,17 +14,22 @@ well-labeled repository of interoperable datasets.
 
 ## Developers
 
+### Pre-requisites
+
+1. Install pre-commit: `pre-commit install` or check doc [here](https://pre-commit.com/)
+2. [Install and configure awscli](docs/awscli.md)
+3. [Configure ssh access](https://github.com/chanzuckerberg/single-cell-infra#ssh)
+4. [Install pyenv](https://github.com/pyenv/pyenv). Make sure you choose the most recent python version from `pyenv`. If `pyenv` is already installed on your machine, you might have to delete and reinstall it.
```

Comment:
I'm not sure we should always use the most recent version of python. I think we currently run it on 3.8